### PR TITLE
Fix operator install docs

### PR DESCRIPTION
**Pull request by jooseppi-luna**
# Description
There is a warning to not mess with sidecar values in the [general operator documentation for all drivers](https://dell.github.io/csm-docs/docs/csidriver/installation/operator/#sidecars), but, in PowerFlex, we need the customer add some values to the yaml file (MDM IPs, etc.), and this is not made clear in the [PowerFlex documentation](general operator documentation for all drivers), causing confusion for users. This is resolved in this PR, with clarifications to both the general operator docs and to the PowerFLex operator docs.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [x] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/operator/powerflex.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Kubernetes Operators make it easy to deploy and manage the entire lifecycle of c
           password: <password in base64>
 ```  
   - Create secret for FTP side by using the command `kubectl create -f sdc-repo-secret.yaml`.
-- Optionally, enable sdc monitor by uncommenting the section for sidecar in manifest yaml.
+- Optionally, enable sdc monitor by uncommenting the section for sidecar in manifest yaml. Please note that you will need to edit the value fields under the HOST_PID and MDM fields by filling the empty quotes with host PID and the MDM IPs. Not filling in these values will cause an error, so please leave this commented out if you don't intend on using it.
```
Comment (AronAtDell) [inline content/docs/csidriver/installation/operator/powerflex.md:42]: If it is required to be filled in, then it really should not be listed as "Optionally...."

Comment (jooseppi-luna) [reply content/docs/csidriver/installation/operator/powerflex.md:42]: It's required if you're using the sidecar. The "optionally" refers to the sidecar itself

Comment (AronAtDell) [reply content/docs/csidriver/installation/operator/powerflex.md:42]: Thanks @jooseppi-luna! Is there a way to bold the "Please note..." or "Note:..."

Comment (jooseppi-luna) [reply content/docs/csidriver/installation/operator/powerflex.md]: @AronAtDell thanks for the suggestion! Let me know how the clarification I submitted looks

**Review by JacobGros — DISMISSED** [review]

File: content/docs/contributionguidelines/_index.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ We use GitHub pull requests for this purpose.
 
 ### Branching strategy
 
-CSM documentation portal follows a scaled trunk branching strategy where short-lived branches are created off of the main branch. When coding is complete, the branch is merged back into main after being approved in a pull request code review.
+The CSM documentation portal follows a release branch strategy where a branch is created for each release and all documentation changes made for a release are done on that branch. The release branch is then merged into the main branch at the time of the release. In some situations it may be sufficient to merge a non-release branch to main if it fixes some issue in the documentation for the current released version.
```
Comment (gallacher) [inline content/docs/contributionguidelines/_index.md:50]: These changes have already been made in the main branch.

Comment (jooseppi-luna) [reply content/docs/contributionguidelines/_index.md:50]: Huh, I have no idea how those changes snuck in. I'll fix that

Comment (jooseppi-luna) [reply content/docs/contributionguidelines/_index.md]: All right, fixed

**Review by shanmydell — APPROVED** [review]

**Review by bharathsreekanth — APPROVED** [review]

**Pull request merged**